src/components/Clock.tsx

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable no-console */
+import { Component } from 'react';
+
+type Props = {
+  clockName: string,
+};
+
+type State = {
+  today: Date,
+};
+
+export class Clock extends Component<Props, State> {
+  state = {
+    today: new Date(),
+  };
+
+  timerId = 0;
+
+  componentDidMount() {
+    this.timerId = window.setInterval(() => {
+      this.setState({ today: new Date() });
+
+      console.info(this.state.today.toLocaleTimeString());
+    }, 1000);
+  }
+
+  componentDidUpdate(prevProps: Props) {
+    const { clockName } = this.props;
+
+    if (clockName !== prevProps.clockName) {
+      console.debug(`Renamed from ${prevProps.clockName} to ${clockName}`);
```

Comment:
Nice job!
Please change *console.debug* to *console.info* 
See more information [link](https://developer.mozilla.org/ru/docs/Web/API/Console/debug)

Reply:
Ok 👍